PYME/Acquire/acquiremainframe.py

```diff
@@ -173,9 +173,11 @@ def _check_init_done(self):
             self.time1.Start(500)
     
     def _refreshDataStack(self):
-        if 'vp' in dir(self):
-            if not (self.vp.do.ds.data is self.scope.frameWrangler.currentFrame):
-                self.vp.SetDataStack(self.scope.frameWrangler.currentFrame)
+        try:
+            if not (self.view.do.ds.data is self.scope.frameWrangler.currentFrame):
+                self.view.SetDataStack(self.scope.frameWrangler.currentFrame)
+        except ArithmeticError:

```

Comment:
```suggestion
        except AttributeError:

```

technically my mistake here - faulty autocompletion